docs/index.md

```diff
@@ -98,5 +98,5 @@ Join our [Slack workspace][slack-workspace] | [Testcontainers OSS][testcontainer
 [windows-container-version-compatibility]: https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility
 [testcontainers-dotnet-contributors]: https://github.com/testcontainers/testcontainers-dotnet/graphs/contributors/
 [slack-workspace]: https://slack.testcontainers.org/
-[testcontainers-oss]: https://www.testcontainers.org/
+[testcontainers-oss]: https://java.testcontainers.org/
```

Comment:
```suggestion
[testcontainers-oss]: https://www.testcontainers.org/
```

Wouldn't it be better to use testcontainers.org (.com) as something that is independent of the programming language?

Reply:
I will revert this change. If you think we should use the Java prefix, let me know, then we need to update the repos README.md too.